azext_iot/central/commands_organization.py

```diff
@@ -74,9 +74,32 @@ def create_org(
         cmd=cmd, app_id=app_id, api_version=api_version, token=token
     )
 
-    return provider.create_organization(
+    return provider.create_or_update_organization(
         org_id=org_id,
         org_name=org_name,
         parent_org=parent_org,
         central_dns_suffix=central_dns_suffix,
     )
+
+
+def update_org(
+    cmd,
+    app_id: str,
+    org_id: str,
+    org_name=None,
+    parent_org=None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+) -> OrganizationV1_1_preview:
+    provider = CentralOrganizationProvider(
```

Comment:
Is there any code that will block a user from trying to update org when the provided api version is not 1.1 preview? Is there at least an appropriate error from the API?

Reply:
all unsupported api versions for commands are handled by outer scope so will never reach implementation.
Here's the output when api-version is forced to unsupported version
![image](https://user-images.githubusercontent.com/2270242/149171885-721a3496-98bb-4733-9440-e880b248922c.png)